icees_api/features/knowledgegraph.py

```diff
@@ -161,126 +225,138 @@ def knowledge_graph_edge(
 
 
 def get(conn, query, verbose=False):
-    message = query.get("message", query)
-    query_options = query.get("query_options", {})
-    cohort_id, table, year, cohort_features, size = message_cohort(conn, query_options)
-    maximum_p_value = query["query_options"].get("maximum_p_value", MAX_P_VAL_DEFAULT)
-    filter_regex = query["query_options"].get("regex", ".*")
-    feature = to_qualifiers(query["query_options"]["feature"])
+    try:
```

Comment:
I'd like to find a cleaner way to catch errors than a 100-line-long try..except block. Have we tried FastAPI's exception handler? https://fastapi.tiangolo.com/tutorial/handling-errors/#install-custom-exception-handlers